Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: harbor
-version: 1.2.0-dev
+version: 1.4.0-dev
```

Comment:
do we need to bump it to 2.0.0? I know our helm version is different from Harbor OSS version.

Reply:
@renmaosheng Was wondering the same till I saw ur comment :smile: So, this is for 2.0.0 right?